resources/views/auth/first_visit_manager.html.twig

```diff
@@ -0,0 +1,96 @@
+                                {% if errors.any() %}
+                                    {# Clone alert box for errors. #}
+                                    <div
+                                        data-c-alert="error"
+                                        data-c-radius="rounded"
+                                        role="alert" data-c-margin="bottom(double)">
+                                        <div data-c-padding="half">
+                                            <span
+                                                data-c-heading="h5"
+                                                data-c-font-weight="bold">
+                                                <i aria-hidden="true" class="fas fa-exclamation-circle"></i>
+                                                Uh oh, something doesn't look right.
+                                            </span>
+                                            <p data-c-margin="top(quarter)">
+                                                Please check the following:
+                                            </p>
+                                            <ul>
+                                                {% for error in errors.all() %}
+                                                    <li>{{ error }}</li>
+                                                {% endfor %}
+                                            </ul>
+                                        </div>
+                                    </div>
+                                {% endif %}
+                                <div data-c-grid-item="base(1of1)">
+                                    <p data-c-margin="bottom(normal)">{{ first_visit.explanation }}</p>
+                                </div>
+                                {# Form Inputs #}
+                                <div data-c-grid="gutter middle">
+                                    {# This div loads a React component #}
+                                    <div
+                                        id="gov-dept-and-email-inputs"
+                                        data-c-grid-item="base(1of1)"
+                                        data-departments="{{ departments|json_encode() }}"
+                                        data-prev-department="{{ old('department') }}"
+                                        data-prev-gov-email="{{ old('gov_email' ) }}"
+                                        >
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        {# UI Actions #}
+                        <div data-c-background="white(100)" data-c-padding="tb(normal) rl(double)" data-c-border="top(thin, solid, grey)">
+                            <div data-c-grid="middle">
+                                <div data-c-grid-item="base(1of2)" data-c-align="base(right)">
```

Comment:
Continue button needs to be centered.
Try this: `<div data-c-grid-item="base(1of1)" data-c-align="base(center)">`